langserver/ast.go

```diff
@@ -59,7 +59,7 @@ func goRangesToLSPLocations(fset *token.FileSet, nodes []*ast.Ident) []lsp.Locat
 
 func goRangeToLSPLocation(fset *token.FileSet, pos token.Pos, end token.Pos) lsp.Location {
 	return lsp.Location{
-		URI:   "file://" + fset.Position(pos).Filename,
```

Comment:
On Windows, file URL uses triple slash, see https://en.wikipedia.org/wiki/File_URI_scheme#Windows
I'm trying to use the same function everywhere